### PR TITLE
Feature [CON-336]: Be able to search by criteria alone, without a search text

### PR DESCRIPTION
**Pull request by habemuscode**
**Dependant issue**

https://github.com/oat-sa/tao-core-ui-fe/pull/244

**Issue**

https://oat-sa.atlassian.net/browse/CON-336

**Description**

**As** an OAT user
I **would** like to be able to search by any combination of existing resource properties (as criteria)
without searching for a default text within those resource properties
**so that** I am still able to find resources for which I don't know the text

**Example**

The user should be able to initiate a search with the following criteria

imagen

**AC**

- Be able to search for any resource (items, tests, test-takers, groups, deliveries, assets) only by property value defined as criteria, without including a text in the search field
- Be able to search for any resource (items, tests, test-takers, groups, deliveries, assets) by any combination of properties values defined as criteria, with or without including a label
- A search can be initiated when at least the search text OR valid search criteria are defined

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "@oat-sa/tao-core-libs": "0.4.2",
         "@oat-sa/tao-core-sdk": "1.7.0",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.20.2",
+        "@oat-sa/tao-core-ui": "git://github.com/oat-sa/tao-core-ui-fe#feature/CON-336/search-by-criteria-without-text",
```
Comment (adriloma) [inline views/package.json:16]: should not wait and be `1.21.0` once [this](https://github.com/oat-sa/tao-core-ui-fe/pull/244) is merged?

Comment (habemuscode) [reply views/package.json:16]: Sorry I forget to put in draft, I will update now

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**